### ISSUE TITLE
[1.3 beta.16] Isobuild/Import scanner is failing when importing a JS file with BOM

### ISSUE THREAD

**Issue by M4v3R** [meteor/meteor#6489]
I've encountered an issue when added `c3` npm package to a project. Here's a basic reproduction repo:

https://github.com/M4v3R/meteor-1.3-bom-bug

To run:

```
(clone the repo)
npm install
meteor
```

Result:

```
TypeError: Object ��//Tree traversing
... code ...
exports.isElementNode = function (node) {
  return !!node.tagName;
}; has no method 'replace'
    at Object.js [as .js] (/tools/isobuild/import-scanner.js:46:23)
```

The offending BOM mark is in `node_modules/c3/node_modules/d3/node_modules/jsdom/lib/jsdom/browser/documentAdapter.js` file.

**Comment by M4v3R** [comment]
Added a pull request that fixes the issue for me.

**Comment by M4v3R** [comment]
For what's it worth, this issue still exists on `rc.1`.

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.4`](https://github.com/meteor/meteor/issues/6266#issuecomment-198997482).
